src/thermopro_ble/parser.py

```diff
@@ -63,6 +71,46 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
         if len(data) < 6:
             return
 
+        if name.startswith("TP96"):
+            MAX_BAT = 2880
```

Comment:
We need to make sure we set the device_type or discovery won't work right

Reply:
Hey @bdraco , I'm still calling `set_device_type` (lines 51/52), I just pulled the value of the parameter out into the `model` variable so I could use it later in the code as well.